src/addrdb.h

```diff
@@ -95,4 +95,10 @@ class CBanDB
     bool Read(banmap_t& banSet);
 };
 
+/** Read the anchor IP address database (anchors.dat) */
```

Comment:
nit: would have been nice to add a little more context information on the purpose of these anchors

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-636453433).